packages/superset-ui-chart-controls/src/index.ts

```diff
@@ -32,3 +32,4 @@ export * from './components/MetricOption';
 // React control components
 export * from './components/RadioButtonControl';
 export * from './types';
+export { default as __hask_reexport__ } from './types';
```

Comment:
Small nit/questions here, @ktmud:

I haven't run into this underscore convention in Superset before. Is there a new pattern emerging here?

If this isn't being imported anywhere, do we need to export it here? 

What is "hask"?
... maybe I'm just tired, but this line makes me feel like I've missed a memo :D

Reply:
It's a typo, this is a _hack_ for `npm link @superset-ui/chart-controls` to work. Somehow it started breaking (probably because of a recent TypeScript update).

cc @zhaoyongjie